### PR TITLE
Add summative to Mini and rename assessment types in Mini and not New…

### PR DESCRIPTION
**Pull request by jtreuting**
… Subject in prep for UAT

### CODE REVIEW

File: deploy/new_subject_config.xml [hunk 1]
```diff
@@ -28,14 +28,14 @@
         </PerformanceLevels>
       </ClaimScoring>
     </AssessmentType>
-    <AssessmentType code="iab" name="Teaching">
+    <AssessmentType code="iab" name="IAB" longName="Interim Assessment Blocks">
```
Comment (malaffoon) [inline deploy/new_subject_config.xml]: should that be plural? i.e. should it be `Block`?

Comment (jtreuting) [reply deploy/new_subject_config.xml]: i copied that from the Math and ELA.  I think it is used in 1 particular place right now where the plural is what is used.  @wtritch ?

Comment (wtritch) [reply deploy/new_subject_config.xml]: yessir, it's plural where it's used.  I believe either singular or plural would work grammatically however:
"Grade X Interim Assessment Blocks Report" and just "Interim Assessment Blocks Report"

File: deploy/new_subject_config.xml [hunk 2]
```diff
@@ -1216,7 +1216,7 @@
       </Targets>
     </Claim>
     <Claim code="CLAIM_6" name="Claim Six" displayOrder="6">
-      <Icon>fa-balance-scale</Icon>
+      <Icon>fa-pencil-square-o</Icon>
```
Comment (wtritch) [inline deploy/new_subject_config.xml]: If you want to use the default, you could just remove this element =)

**Pull request merged**